src/autoplex/fitting/common/jobs.py

```diff
@@ -57,9 +59,11 @@ def machine_learning_fit(
         Use the glue.xml core potential instead of fitting 2b terms.
     glue_file_path: str
         Name of the glue.xml file path.
+    gpu_identifier_indices: list[int]
+        List of GPU indices to be used for fitting. Only used for NEP fitting.
```

Comment:
Only an optional suggestion to keep the docstrings more consistent:
```suggestion
        Use the glue.xml core potential instead of fitting 2b terms. Only used for GAP fitting
    glue_file_path: str
        Name of the glue.xml file path. Only used for GAP fitting
    gpu_identifier_indices: list[int]
        List of GPU indices to be used for fitting. Only used for NEP fitting.
```
and the same for `autodelta` (which I cannot include in the suggestion feature)

Reply:
didnt see the automerge enabled ^^